wallet/chainntfns.go

```diff
@@ -822,10 +822,10 @@ func (w *Wallet) VoteOnOwnedTickets(winningTicketHashes []*chainhash.Hash,
 		addrmgrNs := dbtx.ReadBucket(waddrmgrNamespaceKey)
 		for i, ticketHash := range ticketHashes {
 			ticketPurchase, err := w.TxStore.Tx(txmgrNs, ticketHash)
-			if err != nil || ticketPurchase == nil {
+			if errors.Is(errors.NotExist, err) && ticketPurchase == nil {
```

Comment:
this can be simplified to `if err != nil {`.  The previous code was handling all other errors, and considering the tx not being found also an error.